### PR TITLE
Trap entity

### PR DESCRIPTION
**Pull request by nazariizahoroda**

### CODE REVIEW

File: DeratControl.Domain/Entities/Trap.cs [hunk 1]
```diff
@@ -1,10 +1,19 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using DeratControl.Domain.Root;
 
 namespace DeratControl.Domain.Entities
 {
-    class Trap
+    class Trap : EntityBase<int>
```
Comment (SviatoslavMykhailiv) [inline DeratControl.Domain/Entities/Trap.cs]: should be public visible

File: DeratControl.Domain/Entities/Trap.cs [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
     {
+        public string Data { get; set; }
+        public virtual Point TrapPoint { get; set; }
+        public string TrapType { get; set; }
```
Comment (SviatoslavMykhailiv) [inline DeratControl.Domain/Entities/Trap.cs:14]: please review the DeratControl Requirements.pdf and find possible trap types, think of using some custom type for types, maybe a separate entity or value object

**Review by SviatoslavMykhailiv — APPROVED** [review]
looks good

File: DeratControl.Domain/Entities/Trap.cs (hunk 2 above)
Comment (nazariizahoroda) [reply DeratControl.Domain/Entities/Trap.cs:14]: What type should be data?
I've choosen enum to TrapType

**Pull request merged**